packages/design-system/src/components/toolbar-toggle.tsx

```diff
@@ -0,0 +1,48 @@
+import { styled, theme } from "../stitches.config";
```

Comment:
Please add a header similar to https://github.com/webstudio-is/webstudio-designer/blob/9726664bd8dd0f4c62f9605f075bf9564e2abff0/packages/design-system/src/components/button.tsx#L1-L4